src/sentry/api/endpoints/project_details.py

```diff
@@ -74,6 +75,7 @@ def clean_newline_inputs(value):
 class ProjectMemberSerializer(serializers.Serializer):
```

Comment:
TODO: make ProjectAdminSerializer inherit ProjectMemberserializer and remove dupe code